peps/pep-0771.rst

```diff
@@ -500,6 +463,56 @@ would then get the same as:
 
     $ pip install astropy[recommended, jupyter]
 
+Packages with multiple kinds of defaults
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some cases, it may be that packages need multiple kinds of defaults. As an example,
+in `Packages requiring at least one backend or frontend`_, we considered the case of packages
+that have *either* backends or frontends, but in some cases, packages may have to support
+backends *and* frontends, and want to specify a default frontend and a default backend.
```

Comment:
Or may need multiple backends or frontends. I could see someone having multiple backends available for visualization libraries and giving users access to all through their GUI.

Reply:
I agree, I've tried to reword this by saying 'one or more default frontend and one or more default backend'